docs/api/browser-window.md

```diff
@@ -664,6 +664,13 @@ the player itself we would call this function with arguments of 16/9 and
 are within the content view--only that they exist. Just sum any extra width and
 height areas you have within the overall content view.
 
+#### `win.previewFile(pathname[, filename])` _macOS_
+
+* `pathname` String - The absolute path to the file to preview with QuickLook. This is important as Quicklook uses the file name and file extension on the path to determine the content_type of the file to open.
```

Comment:
Other APIs usually refer to the path as `path` instead of `pathname`, might be good to call this `path` instead for consistency.